### ISSUE TITLE
No Sound

### ISSUE THREAD

**Issue by hkight** [Monika-After-Story/MonikaModDev#5054]
For some reason, I opened the app a few days ago, and I am getting no sound output. I don't know if I messed up any settings, but there's no menu sounds, no rain sounds, no music... Anyone else get this issue before?

**Comment by multimokia** [comment]
Is the `Mute all` checkbox checked in the settings page? (Does it have an X to the left of it?)

**Comment by hkight** [comment]
No, it's not.

**Comment by hkight** [comment]
I think I might have messed up a computer setting. Last I remembered it working properly, I was using a program called Borderless Gaming to try and be able to tab out of fullscreen, but it didn't work properly. I disabled it, and even tried uninstalling the program, but the audio's muted no matter what. Tried playing base DDLC and the sound works fine. I have a feeling it's a computer problem, but I'm not sure what.
